main.cpp

```diff
@@ -96,24 +96,33 @@ good to go!
  struct FloatType
  {
     float add(float lhs, float rhs);
+
     float subtract(float lhs, float rhs);
+
     float multiply(float lhs, float rhs);
+
     float divide(float lhs, float rhs);
  };
 
  struct DoubleType
  {
     double add(double lhs, double rhs);
+
     double subtract(double lhs, double rhs);
+
     double multiply(double lhs, double rhs);
+
     double divide(double lhs, double rhs);
  };
 
 struct IntType
 {
     int add(int lhs, int rhs);
+
     int subtract(int lhs, int rhs);
+
     int multiply(int lhs, int rhs);
+
     int divide(int lhs, int rhs);
 };
```

Comment:
You did not need to add blank lines between each function declaration
- [ ] revise